20_kata.py

```diff
@@ -0,0 +1,15 @@
+# https://www.codewars.com/kata/55c6126177c9441a570000cc
+
+import re
+
+
+def order_weight(strng):
+
+    cleaned_string = re.findall(r"\d+", strng)
+    cleaned_string = sorted(cleaned_string)
```

Comment:
You could have done it in the line above as well: `sorted(re.findall(r"\d+", strng)`, but overall this line is not needed at all. You sort everything at the very end in line 15 and the order of the `cleaned_string` doesn't really matter at this point. Please delete it.

Reply:
It will merged into one line. It needs to be done here. Due of requirement of this exercises, without this sorting in case on same "weight" will be incorrect. 
"When two numbers have the same "weight", let us class them as if they were strings (alphabetical ordering) and not numbers:"